### ISSUE TITLE
Gallery Thumbnail (left/right) Scroll Performance Android Chrome Sluggish and Unresponsive

### ISSUE THREAD

**Issue by markusvonplunkett** [magento/magento2#21014]
### Preconditions (*)

1. Magento 2.2.7 and Magento 2.3 Product Gallery
2. Product page gallery with sample data is installed on the luma theme with additional images (minimum 5+).

### Steps to reproduce (*)

1. Configure product gallery to use thumbnails on mobile.
```
<var name="breakpoints">
    <var name="mobile">
        <var name="conditions">
            <var name="max-width">767px</var>
        </var>
        <var name="options">
            <var name="options">
                  <var name="arrows">true</var>
                  <var name="navarrows">true</var>
                  <var name="nav">thumbs</var>
             </var>
       </var>
   </var>
</var>
```
2. Navigate to a product page with more than 5 images on Chrome Android or Chrome Dev in mobile mode.
3. Attempt to scroll through the *thumbnail* gallery and you will note sluggish/slow performance.

### Expected result (*)

1. Smooth responsive behaviour
2.

### Actual result (*)

1. Sluggish and unresponsive

### Additional information
1. Video of poor performance: https://www.dropbox.com/s/fjjnxsiohgsn8mf/20190206_095707.mp4?dl=0

**Comment by magento-engcom-team** [comment]
Hi @mplunkett5. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@mplunkett5 do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [x] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

-  _Next steps are available in case you are a member of **Community Maintainers**._

- [x] 6. Add label `Issue: Confirmed` once verification is complete. 

- [x] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by ghost** [comment]
Hi @mplunkett5 
The GitHub issue tracker is intended for Magento Core technical issues only.
Please refer to the [Community Forums](https://community.magento.com/) or the [Magento Stack Exchange](https://magento.stackexchange.com/) site for advice or general discussion about this issue.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-backlog-nazar 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-98115`, `MAGETWO-98116` were created

**Issue Available**: @engcom-backlog-nazar, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by slavvka** [comment]
Hi @mplunkett5. Thank you for your report. 
The issue has been fixed in magento/magento2#25839 by @iGerchak in 2.4-develop branch
Related commit(s):
 - [3e57f87910bdfb82ea3ed1ade93e371439b6780b](https://github.com/magento/magento2/commit/3e57f87910bdfb82ea3ed1ade93e371439b6780b)
 - [42a02d5458d49a076cfb8f89daa1a9a23b2199d0](https://github.com/magento/magento2/commit/42a02d5458d49a076cfb8f89daa1a9a23b2199d0)

The fix will be available with the upcoming 2.4.0 release.
